src/config.ts

```diff
@@ -40,7 +42,17 @@ export const configConstants = {
     stage: "dev",
     prefix: "sls",
     localPort: 7071,
+    os: FunctionAppOS.WINDOWS,
   },
+  dockerImages: {
+    node: {
+      "8": "DOCKER|microsoft/azure-functions-node8:2.0",
+      "10": "DOCKER|microsoft/azure-functions/node:2.0"
```

Comment:
is this suppose to be `azure-functions-node`?

Reply:
No, that's the right name... For some reason they decided to change the naming convention that they used for the image when they moved to node 10... Had to look it up on docker hub because it's nowhere in the docs.